srcpkgs/seatd/files/seatd/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/seatd -g video
```

Comment:
This should probably source a conf file, right? So people can enable other `-g` options, if we get more of them.

Reply:
not sure what the best option is here, and it may change as seatd matures. The `-g` flag sets the group which owns seatd's socket. The permissions of this socket determine which users are allowed access to seatd's capabilities. The `video` group is a reasonable default in my opinion, but we could also go with a seatd-specific group and ask people to add themselves to it in the install message.

Reply:
Oh, I see! I would be in favor of a seatd group, I think.